ussodischarge/client.go

```diff
@@ -79,6 +81,22 @@ func (i *Interactor) Interact(ctx context.Context, client *httpbakery.Client, lo
 	return resp.DischargeToken, errgo.Mask(err)
 }
 
+func (i *Interactor) LegacyInteract(ctx context.Context, client *httpbakery.Client, location string, visitURL *stdurl.URL) error {
```

Comment:
Docstring and test for this new method?

Reply:
I added a doc comment, but testing it seems like a lot of work for the amount of extra reassurance it'll give us (not much, given that we can't test against the real idm server and that the code is pretty small).

Code has been QA'd - it does actually work.